scripts/build/Dockerfile.openj9-alpine

```diff
@@ -0,0 +1,35 @@
+FROM adoptopenjdk/openjdk8-openj9:alpine
+
+RUN apk update && apk add \
+	bash \
+	build-base \
+	ccache \
+	coreutils \
+	git \
+	gnutls-dev \
+	libaio-dev \
+	libcap-dev \
+	libnet-dev \
+	libnl3-dev \
+	pkgconfig \
+	protobuf-c-dev \
+	protobuf-dev \
+	python \
+	sudo \
+	maven
+
+COPY . /criu
+WORKDIR /criu
+
+RUN apk add \
```

Comment:
Why are those packages not installed in the previous run of `apk add`?

Reply:
I have changed it. All the packages are now installed together.